mysql/mysqldump_test.go

```diff
@@ -90,6 +91,8 @@ func TestProcessMySQLDump_SingleCol(t *testing.T) {
 
 func TestProcessMySQLDump_MultiCol(t *testing.T) {
 	// Next test more general cases: multi-column schemas and data conversion.
+	numericData := new(big.Rat)
+	numericData.SetString("5.44444")
```

Comment:
nit; if we don't expect an error here (and don't need to catch the same), then it may make sense to use `numericData.SetString("5.44444")` directly where it's used since the return type is big.Rat itself

Reply:
Yes, but numericData.SetString returns two values, so can't use directly.

But you are right, this is kind of messy/ugly. Added mkRat helper function instead (and it checks for failed conversion and triggers test failure if it happens).